### PR TITLE
docs: Document the `enumerate` endpoint

### PR DESCRIPTION
**Pull request by nlsun**

### CODE REVIEW

File: docs/docs/http.md [hunk 1]
```diff
@@ -72,3 +73,64 @@ curl http://10.190.238.173:8123/v1/services/_nginx._tcp.marathon.mesos.
...
+## `GET /v1/enumerate`
+
+Lists in JSON format all DNS information.
+
+```console
+curl http://master.mesos:8123/v1/enumerate
+{
+  "frameworks": [
+   {
+    "tasks": null,
```
Comment (jdef) [inline docs/docs/http.md]: too bad we don't omit this when empty

Comment (nlsun) [inline docs/docs/http.md]: Should I change this? I just copied this out of the DC/OS docs.

Comment (jdef) [inline docs/docs/http.md]: probably fine to have copied. omitting the `null` fields is something that should be configured at in the Golang source.

Comment (sargun) [inline docs/docs/http.md]: I thought we fixed this and it returns []?

Comment (nlsun) [inline docs/docs/http.md]: I went ahead and ran the command on a live cluster and updated it accordingly.

Comment (jdef) [inline docs/docs/http.md]: :+1:

**Comment by jdef** [comment]
lgtm

**Pull request merged**